Source/Core/Core/HW/EXI/EXI_Device.h

```diff
@@ -19,6 +19,7 @@ enum TEXIDevices : int
   EXIDEVICE_AD16,
   EXIDEVICE_MIC,
   EXIDEVICE_ETH,
+  EXIDEVICE_ETHXLINK,
   // Was used for Triforce in the past, but the implementation is no longer in Dolphin.
   // It's kept here so that values below will stay constant.
```

Comment:
Since this comment indicates that values below should "stay constant", I wonder if we should move the entry lower _or_ rethink/reword the comment (which IMO is the more likely candidate). I don't think we store this value anywhere in a config (so that it actually _has to be constant_), but I might be wrong.
Maybe @JosJuice can shed some light on the comment?

Reply:
We are storing it in the config: https://github.com/dolphin-emu/dolphin/blob/a660033e8c394634a22c02a031969685fcdb4fba/Source/Core/Core/ConfigManager.h#L227

So please put the new entry at the bottom.